deploy/template.yaml

```diff
@@ -222,7 +227,7 @@ Mappings:
       may2025RebrandEnabled: true
       alwaysShowBanners: false
       ga4Disabled: false
-      uaDisabled: false
```

Comment:
Is this removal intentional? I think CloudFormation will complain about it being missing